src/assembly_checker_main.cpp

```diff
@@ -144,29 +135,30 @@ int main(int argc, char** argv)
     try {
         auto vcf_path = vm[ebi::vcf::INPUT].as<std::string>();
         auto fasta_path = vm[ebi::vcf::FASTA].as<std::string>();
+        auto assembly_report = vm[ebi::vcf::ASSEMBLY_REPORT].as<std::string>();
         auto fasta_index_path = fasta_path + ".fai";
```

Comment:
I just realised, we can extract this to the constants file, and change the help message, which actually is outdated with "idx"